exporter/datadogexporter/factory.go

```diff
@@ -67,7 +79,13 @@ func enableZorkianMetricExport() error {
 	return featuregate.GlobalRegistry().Set(metricExportNativeClientFeatureGate.ID(), false)
 }
 
-const metadataReporterPeriod = 30 * time.Minute
+const (
+	metadataReporterPeriod = 30 * time.Minute
+	// logSourceName specifies the Datadog source tag value to be added to logs sent from the Datadog exporter.
+	logSourceName = "OTLP log ingestion"
```

Comment:
change this ?

Reply:
Datadog agent OTLP ingest uses the same log source name, which makes sense to me because this is the Datadog source tag value. Both ingest paths fall under "OTLP log ingestion", and `otelSource` further differentiates the two paths.